tailwind.config.js

```diff
@@ -42,41 +50,127 @@ View the full documentation at https://tailwindcss.com.
 | "secondary", or even a numeric scale like "100" and "200", go for it.
 |
 */
-const chroma = require('chroma-js');
-
-let brand = {
-  'primary': '#90278e',
-  'secondary': '#2b388f',
-  'tertiary': '#009bb8',
-}
 
 let colors = {
-  'transparent': 'transparent',
+  ...flat(theme.colors.state, { delimiter: '-' }),
+  ...theme.colors.brand,
+  transparent: 'transparent',
 
-  'black': '#22292f',
+  black: '#22292f',
   'grey-darkest': '#3d4852',
   'grey-darker': '#606f7b',
   'grey-dark': '#8795a1',
-  'grey': '#b8c2cc',
+  grey: '#b8c2cc',
   'grey-light': '#dae1e7',
   'grey-lighter': '#f1f5f8',
   'grey-lightest': '#f8fafc',
-  'white': '#ffffff',
-
-  ...brand,
-
-  'info': chroma(brand.tertiary).brighten(1).css(),
-  'error-dark':'#d8202f',
-  'error-light': '#fadfe1',
-  'warn': chroma(brand.primary).brighten(1).css(),
-  'highlight': '#e83a9c',
-  'active': '#00afed',
-  'inactive': '#dedfe4',
-  'optionSelected': '#e5f6fd'
-}
+  white: '#ffffff',
+
+  'red-darkest': '#3b0d0c',
```

Comment:
Do you want to keep all the tailwind colors here?
Maybe better to replace with our own and use chroma if we want shades too.